doc/02-echo/index.md

```diff
@@ -428,3 +428,11 @@ and informs us at the end of the test. Each of Maelstrom's
 [workloads](/doc/workloads.md) uses different kinds of operations, and checks
 different kinds of properties on them. We'll see additional workloads in later
 chapters.
+
+### Clojure implementations
+
+If you want to run the `clojure` implementation of the echo server, first install [babashka](https://github.com/babashka/babashka) and then run
+
+```
+$ ./maelstrom test -w echo --bin demo/clojure/echo.clj --time-limit 10
+```
```

Comment:
What do you think about putting a `readme.md` in the `demo/clojure` directory with an overview of what each implementation does? That way we don't have to update each section of the guide with a link to every implementation in each language. :)